Samples/HTML2PDFTest/PHP/HTML2PDFTest.php

```diff
@@ -109,28 +98,22 @@
     
 	$settings = new WebPageSettings();
 	$settings->SetZoom(0.5);
-	$converter->InsertFromURL($host.$page0, $settings);
-	$is_conversion_0_successful = $converter->Convert($doc);
+	$converter->InsertFromURL($host, $settings);
+	$converter->Convert($doc);
 
 	//convert page 1 with the same settings, appending generated PDF pages to doc
-	$converter->InsertFromURL($host.$page1, $settings);
-	$is_conversion_1_successful = $converter->Convert($doc);
+	$converter->InsertFromURL($host, $settings);
+	$converter->Convert($doc);
```

Comment:
We should follow Alisher's example I think and not use the same page for every call.

Reply:
That's a good going, variety is added. Thanks!

Reply:
Alisher's URLs don't look that great, because they navigate to the main site, which just shows a blue welcome screen and to accept the cookies otherwise it won't load the rest of the page.

Reply:
I've found 3 URLs that look acceptable in html2pdf:
https://docs.apryse.com/
https://docs.apryse.com/all-products/
https://docs.apryse.com/documentation/web/faq

Everything else is just 33 copies of Welcome to Apryse and Accept cookies Yes No.

Reply:
Sounds good.